events-application-rest-api/pom.xml

```diff
@@ -54,7 +54,6 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>9.4-1203-jdbc42</version>
```

Comment:
I think a version number is also necessary for the dependencies 🤔 Maybe it's better if we put it back, just to be sure 🤓